docs/en/writing-running-appium/touch-actions.md

```diff
@@ -85,7 +85,7 @@ MultiAction().add(action0).add(action1).perform()
 
 ### Bugs and Workarounds
 
-An unfortunate bug exists in the iOS 7.0 - 8.x Simulators where ScrollViews,
+An unfortunate bug exists in iOS Device 7.0 - 8.x Simulators where ScrollViews,
```

Comment:
Maybe something like `bug exists in versions 7.0 to 8.x of iOS (Simulators only)`

Reply:
I think because the wording didn't specifically point out that devices where the culprit is where I failed in the reading. I kept reading iOS and kept thinking it was a bug in the OS on simulators; not the *simulated device*. Seems silly now because that would be a really old OS version and the format of the number could infer it is a device we are talking about. But yet, I still didn't infer that in the moment which still supports my point. How about `bug exists in versions 7.0 to 8.x of iOS Devices (Simulator only)`?

Reply:
> devices is a bit confusing too because device usually implies a real device

yeah....I guess I could agree with that. Probably something close to your original suggestion is the best that can be hoped for: `bug exists in versions 7.0 to 8.x of iOS Simulators`

Reply:
`bug exists in iOS Simulators running on Xcode versions 7.0 to 8.x`